pyvolley.pyw

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env python
 from __future__ import division, print_function
-import ConfigParser
+try:
+    import ConfigParser
+except ModuleNotFoundError:
```

Comment:
`ModuleNotFoundError` was introduced only in Python 3.6, so older versions, like Python 3.5 will throw exception here.

Reply:
You are totally right, sorry. I wrote this small sample:

```
try: 
    import configparser
    print ("ho")
except ImportError: 
    print ("hi")
```

I both executed it locally (Python 3.6) and remotely with Python 2.7 at https://www.tutorialspoint.com/execute_python_online.php at this version seems fine for me